server/lib/pbench/__init__.py

```diff
@@ -153,16 +154,25 @@ def get_pbench_logger(caller, config):
         except FileExistsError:
             # directory already exists, ignore
             pass
-        fh = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
-        fh.setLevel(logging.DEBUG)
+
+        if config.logger_type == "file":
+            handler = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
+        elif config.logger_type == "devlog":
+            handler = logging.SyslogHandler(address='/dev/log', facility=logging.handlers.SysLogHandler.LOG_DAEMON, socktype=socket.SOCK_STREAM)
```

Comment:
Should the facility be really set here? Why not leave it out and let it default to LOG_USER? I'm not sure I understand why this case is different. Can you explain?

Reply:
This helps to log messages to /var/log/daemon.log . This can be removed according to requirement.